### PR TITLE
Migrate to native datetime picker

### PR DESCRIPTION
**Pull request by ChristophWurst**
- [x] Requires https://github.com/nextcloud/nextcloud-vue/pull/3063

### Convert email into event

Bildschirmfoto vom 2022-09-01 13-08-13

### Send later

Bildschirmfoto vom 2022-09-01 13-05-37

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Updated the branch. Both picker still work :+1:

File: src/components/EventModal.vue [hunk 1]
```diff
@@ -6,15 +6,15 @@
 				<input v-model="eventTitle" type="text">
 			</div>
 			<div class="dateTimePicker">
-				<DatetimePicker v-model="startDate"
+				<DateTimePickerNative v-model="startDate"
 					:format="dateFormat"
 					:clearable="false"
 					:minute-step="5"
 					:show-second="false"
```
Comment (JuliaKirschenheuter) [inline src/components/EventModal.vue:13]: this attributes should be removed / replaced. Native datetime picker does't have some of them / or have other analogs

**Comment by JuliaKirschenheuter** [comment]
@ChristophWurst there are some other things to do, i can take over on Monday and finish it

**Comment by ChristophWurst** [comment]
> @ChristophWurst there are some other things to do, i can take over on Monday and finish it

Let's do from scratch when we have time.

**Pull request closed without merging**